SPDX-License-Identifier: GPL-3.0-or-later
Fix compiler error on newer C compilers.
--- sed-4.8.orig/lib/obstack.c
+++ sed-4.8/lib/obstack.c
@@ -326,7 +326,7 @@ int obstack_exit_failure = EXIT_FAILURE;
 #   include <libio/iolibio.h>
 #  endif
 
-static _Noreturn void
+static __attribute_noreturn__ void
 print_and_abort (void)
 {
   /* Don't change any of these strings.  Yes, it would be possible to add
